src/applications/disability-benefits/all-claims/Form526EZApp.jsx

```diff
@@ -30,7 +37,27 @@ export const hasRequiredServices = user =>
 export const hasRequiredId = user =>
   idRequired.some(service => user.profile.services.includes(service));
 
-export function Form526Entry({ location, user, children, mvi }) {
+export const getWizardStatus = () =>
+  sessionStorage.getItem(WIZARD_STATUS) || WIZARD_STATUS_NOT_STARTED;
+
+export const Form526Entry = ({ location, user, children, mvi, showWizard }) => {
+  const defaultWizardState = getWizardStatus();
+  const [wizardState, setWizardState] = useState(defaultWizardState);
+
+  const setWizardStatus = value => {
+    window.sessionStorage.setItem(WIZARD_STATUS, value);
+    setWizardState(value);
+  };
+
+  useEffect(() => {
+    if (defaultWizardState === WIZARD_STATUS_COMPLETE) {
+      setWizardStatus(WIZARD_STATUS_COMPLETE);
+    }
+  });
+  if (showWizard && wizardState !== WIZARD_STATUS_COMPLETE) {
+    return <WizardContainer setWizardStatus={setWizardStatus} />;
```

Comment:
So when `showWizard` is `true`, and the `wizardState` is not complete, we render the wizard instead of the normal app, correct?

Reply:
Yes, that was part of the designed flow for how the wizard would work. There is also a link at the bottom of the page to skip the wizard and render the intro page content directly.